src/pkg/bundle/common_test.go

```diff
@@ -160,39 +161,52 @@ func Test_getPkgPath(t *testing.T) {
 		{
 			name: "init full path",
 			args: args{
-				pkg:  types.Package{Name: "init", Ref: "0.0.1", Path: "../fake/path/custom-init.tar.zst"},
-				arch: "fake64",
+				pkg:    types.Package{Name: "init", Ref: "0.0.1", Path: "../fake/path/custom-init.tar.zst"},
+				arch:   "fake64",
+				srcDir: "/mock/source",
 			},
-			want: "../fake/path/custom-init.tar.zst",
```

Comment:
No need to remove the existing tests because those test for existing functionality that we'd like to keep, you'll need to add the absolute path test cases to this list of tests

Reply:
Clearly I didn't read carefully enough 😆